### PR TITLE
Add Twin Primes algorithm

### PR DESCRIPTION
**Pull request by Exortions**
[Open in Gitpod](https://gitpod.io/from-referrer/)&nbsp;[know more](https://www.gitpod.io/docs/pull-requests/)

### Describe your change:

- [x] Add an algorithm?
- [ ] Fix a bug or typo in an existing algorithm?
- [ ] Documentation change?

### Checklist:

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Javascript/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new JavaScript files are placed inside an existing directory.
- [x] All filenames should use the UpperCamelCase (PascalCase) style. There should be no spaces in filenames.
      **Example:**`UserProfile.js` is allowed but `userprofile.js`,`Userprofile.js`,`user-Profile.js`,`userProfile.js` are not
- [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: Maths/TwinPrime.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+function isPrime (n) {
```
Comment (raklaptudirm) [inline Maths/TwinPrime.js:1]: We already have prime checking algorithms, use those instead of duplicating code.

File: Maths/TwinPrime.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+/**
+ * @function twinPrime
+ * Gets the 'twin prime' of a prime number.
+ * @returns {Array} Either an array with the original [0], and the twin [1], or an empty array if one of the numbers are not prime.
+ * @see https://en.wikipedia.org/wiki/Twin_prime
+ * @example twinPrime(5) = [5, 7]
+ * @example twinPrime(4) = []
+*/
+function twinPrime (n) {
+  const result = []
```
Comment (raklaptudirm) [inline Maths/TwinPrime.js:25]: Just return the twin prime, no need for an array.

File: Maths/test/TwinPrime.test.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { twinPrime } from '../TwinPrime.js'
+
+describe('Twin Primes', () => {
```
Comment (raklaptudirm) [inline Maths/test/TwinPrime.test.js:3]: Add more test cases.

**Review by raklaptudirm — APPROVED** [review]

File: Maths/TwinPrime.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+function isPrime (n) {
+  let prime = false
```
Comment (appgurueu) [inline Maths/TwinPrime.js:2]: To add to what rak said, this variable is named misleadingly and the below sieve is pretty poor (O(n)) compared to other algos.

Comment (Exortions) [reply Maths/TwinPrime.js:2]: What's the name of the algorithm already implemented for primes?

Comment (Exortions) [reply Maths/TwinPrime.js:2]: Is it Maths/PrimeCheck?

Comment (raklaptudirm) [reply Maths/TwinPrime.js]: Yeah, you can use that one.

**Review by appgurueu — APPROVED** [review]

**Review by raklaptudirm — APPROVED** [review]

**Pull request merged**